Sources/Plot/API/HTML.swift

```diff
@@ -148,6 +150,9 @@ public extension HTML {
 
 /// Context shared among all HTML elements.
 public protocol HTMLContext {}
+/// Context shared among all HTML elements that support the `datetime`
+/// attribute, such as `<time>`.
+public protocol HTMLDateTimeContext: HTMLContext {}
```

Comment:
This protocol shouldn't be needed, since it's only adopted by a single type. You should be able to use `TimeContext` directly within your `Node` type constraints. Unless I'm missing something?

Reply:
Good point